### DISCUSSION TITLE
Define community.mysql collection mission statement

### DISCUSSION THREAD

**Discussion by Andersson007** [ansible-collections/ansible.mysql#560] [General]
@laurent-indermuehle @eRadical @rsicart @Jorge-Rodriguez @betanummeric @markuman @elpavel 

Hello everyone,

I suggest defining our community mission statement.

A mission statements holds a lot of value for any kind of collaborative project.
Our mission statement is our audacious goal. We could put it in our README, etc.

As a starting point, i suggest the following mission statement:
```
Ansible community.mysql collection community mission is to produce simple, flexible,
and powerful open-source software for automating MySQL- and MariaDB-related tasks.
```

This single sentence IMO communicates the following key aims of the project:
- The kind of tool we wish to produce: automation software
- The primary functions of the tool: automation of MySQL/MariaDB-related tasks
- The principles of the project: simplicity, flexibility, ease of use
- The scope of its approach: using open source technology / the mentioned RDBMSs

**Goals**. i can name the following on the spot:

- many mature (and not only) projects have their mission statements defined. it shows maturity and understanding what's the goal of the project, serious approach, etc.
- informational: external folks will see what's the core goal of the project and our principles and values like open source, etc.
- motivational: newcomers and we ourselves can revisit it from time to time for inspiration based on our community values:)

What do you folks think? If you're OK with the idea but have better wording on you mind, please suggest

**Comment by laurent-indermuehle** [comment]
I think this is a good idea. But I don't have any opinion on this.

**Reply by Andersson007** [reply]
@laurent-indermuehle thanks for the response! Maybe any wording improvement then or it sounds OK to you as is?

**Comment by Andersson007** [comment]
OK, i'm voting with +1 to the statement as-is:)

**Comment by Andersson007** [comment]
Created a PR https://github.com/ansible-collections/community.mysql/pull/561, could you please take a look?

**Comment by MarkOStewart** [comment]
Slight rewrite. 

**Community Mission:** The Ansible community.mysql collection goals are to produce and maintain simple, flexible,
and powerful open-source software for automating MySQL and MariaDB related tasks and to document the collection for easy deployment and use.

**Reply by Andersson007** [reply]
@MarkOStewart hello, thanks for the feedback!
I like your wording and it's incorporated.
I like that you mentioned the doc. I added a slightly changed version of that part (trying to avoid too many `and`) but I'm not sure if it's OK now.
Could you please take a look at https://github.com/ansible-collections/community.mysql/pull/561 ?
Thank you

**Reply by MarkOStewart** [reply]
Not sure what you want on #561. I looked but not sure what you are asking for?

**Reply by Andersson007** [reply]
@MarkOStewart does it look good to you now?
